unit_tests/utilities/test_zaza_utilities_juju.py

```diff
@@ -22,15 +22,24 @@ class TestJujuUtils(ut_utils.BaseTestCase):
     def setUp(self):
         super(TestJujuUtils, self).setUp()
 
+        class MachineMock(dict):
+
+            def __init__(self, display_name=''):
+                self.display_name = display_name
+
         # Juju Status Object and data
         self.key = "instance-id"
         self.key_data = "machine-uuid"
 
         self.machine1 = "1"
         self.machine1_data = {self.key: self.key_data}
```

Comment:
is this variable still used?

Reply:
It is, some of the tests themselves reference it

Reply:
It is, some of the tests themselves reference it

Reply:
indeed but it looks like `machine2_data` is gone